ts/src/binance.ts

```diff
@@ -10230,7 +10230,6 @@ export default class binance extends Exchange {
         } else {
             collateralString = this.safeString (position, 'isolatedMargin');
         }
-        collateralString = (collateralString === undefined) ? '0' : collateralString;
```

Comment:
what is rationale to be `0` if it's not defined? zero (as a number) is below `maintenanceMargin` amount which means position's collateral "is below" MM, which should be translated at triggering liquidation. so, we shouldnt set that to zero, but to `undefined`